lib/SPIRV/OCLUtil.cpp

```diff
@@ -1335,26 +1335,6 @@ static StringRef getStructName(Type *Ty) {
   return "";
 }
 
-Value *unwrapSpecialTypeInitializer(Value *V) {
```

Comment:
This function is specifically use for non opaque pointers.
Removing it completely and replacing the return value in call-sites with nullptr.